frontend/package.json

```diff
@@ -4,8 +4,14 @@
   "private": true,
   "dependencies": {
     "prop-types": "^15.7.2",
+    "@material-ui/core": "^4.5.2",
+    "bootstrap": "^4.3.1",
+    "missing": "0.0.1",
+    "node-sass": "^4.13.0",
     "react": "^16.11.0",
+    "react-bootstrap": "^1.0.0-beta.14",
```

Comment:
Do you really need both bootstrap and material-ui? Usually one would use only one or the other.